vendor.conf

```diff
@@ -37,7 +37,7 @@ github.com/moby/sys 74ec3fe1325a016151d353bb6654d960042f9e96
 github.com/modern-go/concurrent 1.0.3
 github.com/modern-go/reflect2 1.0.1
 github.com/opencontainers/go-digest c9281466c8b2f606084ac71339773efd177436e7
-github.com/opencontainers/image-spec deb02d24daef68ac4a88d8f9883ba03b75bdc187 https://github.com/kevpar/image-spec.git # fork
+github.com/opencontainers/image-spec v1.0.1
```

Comment:
I think this will need to update `go.mod` as well now that we've moved away from vndr. Is that right @katiewasnothere?

Reply:
Or maybe this also needs to be rebased on the new `windows_port`.